internal/pkg/describe/env_test.go

```diff
@@ -235,26 +300,7 @@ func TestEnvDescription_HumanString(t *testing.T) {
 	}
 	allSvcs := []*config.Service{testSvc1, testSvc2, testSvc3}
 
-	wantedContent := `About
-
-  Name              testEnv
-  Production        false
-  Region            us-west-2
-  Account ID        123456789012
-
-Services
-
-  Name              Type
-  testSvc1          load-balanced
-  testSvc2          load-balanced
-  testSvc3          load-balanced
-
-Tags
-
-  Key               Value
-  key1              value1
-  key2              value2
-`
+	wantedContent := "About\n\n  Name              testEnv\n  Production        false\n  Region            us-west-2\n  Account ID        123456789012\n\nServices\n\n  Name              Type\n  testSvc1          load-balanced\n  testSvc2          load-balanced\n  testSvc3          load-balanced\n\nTags\n\n  Key               Value\n  key1              value1\n  key2              value2\n\nResources\n\n  AWS::IAM::Role           testApp-testEnv-CFNExecutionRole\n  testApp-testEnv-Cluster  AWS::ECS::Cluster-jI63pYBWU6BZ\n"
```

Comment:
can you follow the previous example to show the `wantedContent` in a more human-readable format? That'll also help understanding the expected output.